### ISSUE TITLE
Cycling through Themes using Arrow Keys Without Being in the Settings Window

### ISSUE THREAD

**Issue by koalazub** [warpdotdev/Warp#294]
### Describe the bug

```shell
- Terminal is open
- No windows open that are related to Warp i.e. settings/preferences
- Hit the `up` arrow to navigate to history of commands
- Themes instead are being cycled in the order they're listed in the settings window
i.e. If I'm already in light mode, the `up` arrow takes me to dark
```

### To Reproduce

Steps to reproduce:

- I'm still trying to figure out how that happened
- I think I had just woken up the Mac and shifted from Zoom to Warp

### Expected behaviour

```shell
To be able to cycle through the history of commands from the command line
```

### Screenshots

https://user-images.githubusercontent.com/7111524/138375733-df56e1f3-039f-49a5-8179-555920d779be.mov

This is me cycling through with the `up` and `down` arrows

### Operating System

MacOS

### OS Version

11

**Comment by elviskahoro** [comment]
@zheng Circling back.

**Comment by elviskahoro** [comment]
@koalazub has it happened to you recently? We may close and then just reopen if it comes back. Warp has since gotten an update to the the settings dialog which might have fixed it.

**Comment by koalazub** [comment]
@elviskahoro Agreed. I think there have been 3 updates since this was reported. 

Will close with the option to reopen as you mentioned

**Comment by elviskahoro** [comment]
We were actually able to reproduce it internally so reopening.

**Comment by elviskahoro** [comment]
Fixed! 2022.02.16
